localization/ndt_scan_matcher/src/ndt_scan_matcher_core.cpp

```diff
@@ -193,31 +193,36 @@ NDTScanMatcher::NDTScanMatcher()
 
   z_margin_for_ground_removal_ = this->declare_parameter<double>("z_margin_for_ground_removal");
 
-  rclcpp::CallbackGroup::SharedPtr initial_pose_callback_group;
-  initial_pose_callback_group =
+  timer_callback_group_ = this->create_callback_group(rclcpp::CallbackGroupType::MutuallyExclusive);
+  rclcpp::CallbackGroup::SharedPtr initial_pose_callback_group =
+    this->create_callback_group(rclcpp::CallbackGroupType::MutuallyExclusive);
+  rclcpp::CallbackGroup::SharedPtr sensor_callback_group =
     this->create_callback_group(rclcpp::CallbackGroupType::MutuallyExclusive);
-
-  rclcpp::CallbackGroup::SharedPtr main_callback_group;
-  main_callback_group = this->create_callback_group(rclcpp::CallbackGroupType::MutuallyExclusive);
 
   auto initial_pose_sub_opt = rclcpp::SubscriptionOptions();
   initial_pose_sub_opt.callback_group = initial_pose_callback_group;
-
-  auto main_sub_opt = rclcpp::SubscriptionOptions();
-  main_sub_opt.callback_group = main_callback_group;
-
+  auto sensor_sub_opt = rclcpp::SubscriptionOptions();
+  sensor_sub_opt.callback_group = sensor_callback_group;
+
+  const double map_update_dt = 1.0;
```

Comment:
You can use constexpr here.

Reply:
Fixed in https://github.com/autowarefoundation/autoware.universe/pull/5915/commits/39ec48782c1c3cce99fbf00b836f96ed48e5b7cc